src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  max-height: max-content;
+  width: 1024px;
```

Comment:
I think width may better be defined with the help of variables

Reply:
agree